desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -188,6 +188,10 @@
     public static final String DICTIONARY_TEXTFILE_IMAGEPATH = "images/model/dictionary_textfile.png";
     public static final String DICTIONARY_DATASTORE_IMAGEPATH = "images/model/dictionary_datastore.png";
 
+    public static final String NEWS_CHANNEL = "images/news/news_channel_16.png";
+    public static final String NEWS_CHANNEL_ORANGE = "images/news/news_channel_28.png";
+    public static final String NEWS_CHANNEL_HIGHLIGHTED = "images/news/news_channel_20.png";
```

Comment:
Okay, that's fine, we'll just have to make new ones, then. I guess they should be reasonably easy to copy anyway.

Just a clarification of the resizable icons, there's two main things to icon sizes:
- Simple scalability; Are they using a scalable format like SVG or using something sufficiently high resolution that scaling down shouldn't afffect the quality too much. First one is usually preferred (but we haven't been good at using them anyway).
- The complexity of the image: You can't just scale down something with a lot of small details. Usually what I see is icon designers doing 3 (or more) different versions in varying degrees of complexity. Those would then be scalable separately. I'm guessing that's what your graphics guru was referring to.

Making this much more complex is the issue of DPI: Even though your phone (probably) has an insanely high resolution, you can't just use the complex icon, as it would still look smudgy in small sizes. Currently that isn't an issue for DataCleaner, though, as we don't support DPI scaling in any case (which is actually becoming a problem on new laptops, as they also have insane resolutions, so everything looks _super_ small.)

Whoops, that was a bit of a sidestep... :)